UnityProjects/MRTKDevTemplate/Assets/Scripts/PerfSceneManager.cs

```diff
@@ -9,28 +12,29 @@ namespace Microsoft.MixedReality.Toolkit.Examples.Demos
 {
     /// <summary>
     /// Manager for the performance evaluation scene.
-    /// It instantiates gameobjects until the FPS drops below a set
-    /// limit, currently 50 fps.
     /// </summary>
+    /// <remarks>
+    /// This will instantiate game objects until the frame rate drops below a set limit.
+    /// </remarks>
     public class PerfSceneManager : MonoBehaviour
     {
         // Reference to the description panel.  The description is disabled when the test is run.
         [SerializeField]
         private GameObject descriptionPanel;
 
-        // Reference to the gameobject to spawn.
+        // Reference to the game object to spawn.
```

Comment:
nit - I think that since GameObject is an actual class, this should be a cref or capitalized as GameObject.